### PR TITLE
Some minor fixes to actors examples

### PR DESCRIPTION
**Pull request by BlackMinou**
Tried to follow @tremblay-guy commentaries on #2361  to improve the code of the actors examples

### CODE REVIEW

File: lib/actors/examples/fannkuchredux/fannkuchredux.nit [hunk 1]
```diff
@@ -36,21 +36,14 @@ class FannkuchRedux
...
 
-			for j in [0..i] do
-				if j + d <= i then
-					p[j] = pp[j+d]
-				else
-					p[j] = pp[j+d-i-1]
-				end
-			end
+			for j in [0..i] do p[j] = if j + d <= i then pp[j+d] else pp[j+d-i-1]
 			i -= 1
```
Comment (BlackMinou) [inline lib/actors/examples/fannkuchredux/fannkuchredux.nit]: no ...

File: lib/actors/examples/fannkuchredux/fannkuchredux.nit [hunk 2]
```diff
@@ -36,21 +36,14 @@ class FannkuchRedux
 	fun first_permutation(idx: Int) do
 		for i in [0..p.length[ do p[i] = i
 
-		var i = count.length - 1
-		while i > 0 do
+		for i in [0..(count.length - 1)].reverse_iterator do
```
Comment (BlackMinou) [inline lib/actors/examples/fannkuchredux/fannkuchredux.nit]: lol I'm dumb haha !

File: lib/actors/examples/fannkuchredux/fannkuchredux.nit (hunk 1 above)
Comment (xymus) [inline lib/actors/examples/fannkuchredux/fannkuchredux.nit]: `i` is set at the next iteration of the `for` loop anyway... Or is there something I'm missing?

Comment (BlackMinou) [inline lib/actors/examples/fannkuchredux/fannkuchredux.nit]: wasnt looking at the same line hahahah, you're right

**Pull request merged**